application/associations.py

```diff
@@ -263,7 +263,10 @@ def edit_file(self, selection):
 			self._application.create_terminal_tab(active_object._notebook, options)
 
 		else:
-			subprocess.Popen(split_command, cwd=os.path.dirname(selection[0]))
+			cwd = None
```

Comment:
Provider does have `is_local` (so to speak) constant. This can be used to test such behavior. But I don't think it takes into account `file://` protocol.